integrationtests/tests/fixtures/mod.rs

```diff
@@ -284,7 +285,8 @@ pub async fn fixtures(num_peers: u16) -> anyhow::Result<Fixtures> {
                                 "wallet",
                                 Wallet::new_with_bitcoind(
                                     cfg.get_module_config_typed("wallet").unwrap(),
-                                    fed_db(),
+                                    // wallet module does not need any module specific types
```

Comment:
Ah, I've overlooked it. 

I'm not so sure about it...

It has inputs, outputs, etc.

Reply:
We should pass the `&Database` to the closure here. I'll have this in #1269

Reply:
I think this should work. 
Current this db is only passed to broadcast pending task, which also used default decoders before.